src/DotNetBridge/DotNetBridge.csproj

```diff
@@ -14,6 +14,8 @@
     <Copyright>(c) Microsoft Corporation. All rights reserved.</Copyright>
     <RepositoryUrl>https://github.com/Microsoft/NimbusML</RepositoryUrl>
     <PackageProjectUrl>https://github.com/Microsoft/NimbusML</PackageProjectUrl>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>tmsn.snk</AssemblyOriginatorKeyFile>
```

Comment:
why is this named `tmsn.snk`? that seems cryptic

Reply:
Do you also want to set

```
<PublicSign Condition="'$(OS)' != 'Windows_NT'">true</PublicSign>
```

see
https://github.com/dotnet/machinelearning/blob/master/Directory.Build.props#L105

Reply:
I copied tmsn.snk from TLC to keep the keys the same for ML.NET external friend repos. I'll rename it to something less cryptic.